index.html

```diff
@@ -126,17 +126,17 @@ <h1 class="project-title">Multi-Post Stories</h1>
     <div class="projects-card">
       <div class="card-header">
         <div class="projects">
-          <img src="./Images/Snapshoot Portfolio.png" alt="" class="project-screenshot">
+          <img src="./Images/Snapshoot-Portfolio-1.png" alt="Professional-art-printing" class="project-screenshot">
```

Comment:
Kindly check your image path, your images are not displaying 😊

![image](https://user-images.githubusercontent.com/73854999/200682376-daa85a35-c3d2-4035-af07-e12e9341ab66.png)